curriculum/challenges/english/21-a2-english-for-developers/learn-greetings-in-your-first-day-at-the-office/656cc3b856f1d01011bfc027.md

```diff
@@ -0,0 +1,53 @@
+---
+id: 656cc3b856f1d01011bfc027
+title: Task 94
+challengeType: 19
+dashedName: task-94
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Tom: Wow, I'm so hungry. Is it lunch time?
+-->
+
+# --description--
+In English, the three main meals of the day are breakfast, lunch, and dinner. Breakfast is typically eaten in the morning, lunch is around midday, and dinner is in the evening.
```

Comment:
I think "breakfast", "lunch", and "dinner" should go in backticks in the description and in the feedbacks.